minCostFlow.py

```diff
@@ -49,7 +51,7 @@ def construct_digraph(edges_file, cap):
 
 def print_graph(graph):
     ''' Print the edges in a graph '''
-    print('\n'.join(sorted(map(str, graph.edges(data=True)))))
+    print(('\n'.join(sorted(map(str, graph.edges(data=True))))))
```

Comment:
It looks like this print already had `(...)` and would work with Python 3.  Did it need another set of parentheses?